addOns/formhandler/src/main/java/org/zaproxy/zap/extension/formhandler/OptionsFormHandlerPanel.java

```diff
@@ -185,7 +190,8 @@ public boolean showRemoveDialogue(FormHandlerParamField e) {
 
             if (option == JOptionPane.OK_OPTION) {
                 setRemoveWithoutConfirmation(removeWithoutConfirmationCheckBox.isSelected());
-
+                Stats.incCounter(ExtensionFormHandler.STATS_REMOVE);
+                Stats.incCounter(ExtensionFormHandler.STATS_REMOVE + "." + field.getName());
```

Comment:
Could be extracted a method, e.g. `incStat(ExtensionFormHandler.STATS_REMOVE, field)`.